src/_guides/language/effective-dart/design.md

```diff
@@ -6,7 +6,7 @@ prevpage:
   url: /guides/language/effective-dart/usage
   title: Usage
 ---
-<?code-excerpt replace="/\/\/!//g;/ellipsis;?/.../g;/\/\*(\s*\.\.\.\s*)\*\//$1/g;/\{\/\*-(\s*\.\.\.\s*)-\*\/\}/$1/g;/([A-Z]\w*)\d\b/$1/g"?>
+<?code-excerpt replace="/([A-Z]\w*)\d\b/$1/g"?>
```

Comment:
When we have regexps, could you add a comment explaining them?

Reply:
This one is used in a few files. I've explained it's purpose in the README.

Reply:
To answer your question more directly :) ... yes, I will explain them, most certainly the global ones, unless their intent is obvious (like `/,//g` erases commas). Is that ok?